hub/api/dataset.py

```diff
@@ -167,7 +171,28 @@ def __init__(
             self._schema = hub.schema.deserialize.deserialize(self.meta["schema"])
             self._meta_information = self.meta.get("meta_info") or dict()
             self._flat_tensors = tuple(flatten(self._schema))
+            try:
+                version_info = pickle.loads(fs_map["version.pkl"])
+                self._branch_node_map = version_info["branch_node_map"]
+                self._commit_node_map = version_info["commit_node_map"]
+                self._commit_optimized_map = version_info["commit_optimized_map"]
+                self._chunk_commit_map = version_info["chunk_commit_map"]
+                self._branch = "master"
+                self._version_node = self._branch_node_map[self._branch]
+                self._commit_id = self._version_node.commit_id
+                self._is_optimized = self._commit_optimized_map[self._commit_id]
+            except Exception:
+                self._commit_id = None
```

Comment:
Add other exceptions

Reply:
@imshashank There could be multiple exceptions, while trying to load version_info, getting maps from version_info, and getting the commit_id. In all possible scenarios (the above list might be non-exhaustive), our exceptions should be added in the same way, so I don't feel a need for adding more exceptions here

Reply:
Same regarding exceptions.